### PR TITLE
Updated Forms

### PR DESCRIPTION
**Pull request by MacRichards**
Updated Inscurance Forms

### CODE REVIEW

File: resources/sections.md [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
+date:   2017-04-12 20:10 -0800
 description: "Resources for Sections in the Western Region."
 ---
 
 ## Conclave Insurance Payment Transmittal Form
 
 The transmittal paperwork for the annual Section Conclave's insurance payment. This is a fillable PDF.
 
-<a href="{{ site.baseurl }}resources/assets/2017-conclave-insurance-payment-transmittal-fillable.pdf" class="btn btn-default"><i class="fa fa-file-pdf-o"></i> Download Insurance Payment Transmittal Form</a>
+<a href="{{ site.baseurl }}resources/assets/2017-Conclave-Insurance-Payment-Transmittal-Fillable.pdf" class="btn btn-default"><i class="fa fa-file-pdf-o"></i> Download Insurance Payment Transmittal Form</a>
```
Comment (blove) [inline resources/sections.md]: I think the Linux server that we are on is case-sensitive. Please adjust the case to match the file.

**Comment by blove** [comment]
@MacRichards RE my comment, please make the link case-sensitive. Then I can get this PR merged and put live.

File: resources/sections.md (hunk 1 above)
Comment (MacRichards) [reply resources/sections.md]: The PDF files I created also have upper-case in the file name so they do match.

Comment (MacRichards) [reply resources/sections.md]: Never mind, I upper-cased the file and it didn't upload that way. I fixed the casing.

**Comment by blove** [comment]
Thanks, @MacRichards 👍

**Pull request merged**